lightning/src/ln/channelmanager.rs

```diff
@@ -1701,9 +1701,11 @@ where
 	/// Returns in an undefined order recent payments that -- if not fulfilled -- have yet to find a
 	/// successful path, or have unresolved HTLCs.
 	///
-	/// This can be useful for figuring out whether or not a payment needs to be retried.
-	/// In general, if it is not listed here, you should consider retrying it if not
-	/// [`RecentPaymentDetails::Fulfilled`].
+	/// This can be useful for payments that may have been prepared, but ultimately not sent, as a
+	/// result of a crash. If such a payment exists, is not listed here, and an
+	/// [`Event::PaymentSent`] event has yet to be received, you may consider retrying the payment.
```

Comment:
No need to repeat "event". How about "an [`Event::PaymentSent`] has not been received"?